workshops/hello_bot/README.md

```diff
@@ -0,0 +1,89 @@
+```js
+bot.on("message", (message) => { console.log(message.content); });
+```
+
+There's quite a bit going on in that bit of code. So let's break it down. `bot.on()` is the function that we use to set what function will be called when the specified event is fired. `"message"` tells `discord.js` which event that function is for. In this case it is the `message` event. The `(message) => {}` bit is the lambda method for creating a function, you can read more about those [here](https://www.vinta.com.br/blog/2015/javascript-lambda-and-arrow-functions/). The final part of that block of code is `console.log(message.content`, this logs the message that was sent to the console. At this point your `index.js` file should look like the following:
+```js
+const Discord = require("discord.js");
+cons discord = new Discord.Client();
+bot.on("message", (message) => { console.log(message.content); });
+bot.login("token");
+```
+
+# The Hello World Command
+
+Most Discord bots have a thing called a prefix. The prefix allows the bot to tell the difference between a regular message and a command it is meant to interpret. It is not recommended to use a prefix that is already largely popular, some examples are `!`, `?`, `/`, and `=`. Instead, combine some of those options or use something unique like `-`. For this workshop, we will use `h!` as our prefix. We want to declare a constant for our prefix, so we only have to change it in one place if we decide to update it later. We can do that using the following line:
+```js
+const prefix = "h!";
+```
+
+Now, we will update our listener function to check that the message starts with our prefix. The following is what the updated listener should look like:
+```js
+bot.on("message", (message) => { if(!message.content.startsWith(prefix)) return;
+console.log(message.content); });
+```
+
+This code will check for the prefix, and if the message doesn't start with it it will exit. If it contains the prefix it will print the entire message to the console. However, to get the actual command we will need to remove the prefix from the message string:
+```js
+bot.on("message", (message) => {
+  if(!message.content.startsWith(prefix)) return;
+  var command = message.content.slice(prefix.length);
+  console.log(command);
+});
+```
+
+This code will now only print the command from the message. Now we should actually have the bot send a message in Discord! For this workshop, the command will be `hello` and the response from the bot will be `world`. We can do that by updating the listener to the following:
+```js
+bot.on("message", (message) => {
+  if(!message.content.startsWith(prefix)) return;
+  var command = message.content.slice(prefix.length);
+  if(command == "hello") {
+    message.reply("world");
+  }
+});
+```
```

Comment:
Include the full source code at the end, so people can check their code against yours. Something like "Your whole file should now look like this: <code>"